unity-renderer/Assets/DCLServices/EmotesCatalog/EmotesCatalogBridge.cs

```diff
@@ -52,10 +53,49 @@ public void AddEmotesToCatalog(string payload)
         if (request == null)
             return;
 
-        if (string.IsNullOrEmpty(request.context))
-            OnEmotesReceived?.Invoke(request.emotes);
+        var emote = request.emotes.FirstOrDefault(e => e.id.EndsWith("6d"));
+        if (emote != null)
+        {
+            Debug.LogError("INVALID EMOTE TESTS: AddEmotesToCatalog: Catalog received an emote " + emote.id);
```

Comment:
This seems like a test debug, it shouldnt be pushed.

Reply:
@AjimenezDCL, what do you think of using `Debug.Assert` in such cases, since it will be not included in the built?